### DISCUSSION TITLE
Uncaught (in promise) SyntaxError: Invalid or unexpected token

### DISCUSSION THREAD

**Discussion by dovietthang** [vue-leaflet/vue-leaflet#322] [Q&A]
Error:
Uncaught (in promise) SyntaxError: Invalid or unexpected token (at marker-shadow.png?import:1:1)

mycode:

    <l-map :zoom="zoom" :center="center" :use-global-leaflet="true" :options="{ zoomControl: false }">

      <l-tile-layer :url="tileLayerUrl" :attribution="attribution" layer-type="base"></l-tile-layer>
      <l-control-zoom position="bottomright"></l-control-zoom>
      <l-marker v-for="(map, key) in dataMap" :key="key" :lat-lng="map.markerLatLng" @click="handleClickMap(map)">
        <l-tooltip :options="{ interactive: false, permanent: false, direction: 'top', opacity: 1 }">
          <div>
            <span class="my-0 mx-5 fs-5 text-primary fw-bold d-flex align-items-center x-index-2">
              <i class="fa-solid fa-plug-circle-plus me-2" style="color: #163c7e;"></i>
              Connect to {{ map.title }}
            </span>
          </div>
        </l-tooltip>
        <l-icon :icon-size="[32, 32]" :icon-anchor="[16, 16]" :icon-url="iconUrl" className="">
          <i class="fa-regular fa-circle-dot fs-2hx" style="color: #1e53ae;"></i>
        </l-icon>
      </l-marker>
    </l-map>

**Comment by veitbjarsch** [comment]
You have no imports. 


Long answer:
Can you please provide a small setup where you reproduce the issue, so we can have a deeper look into it? This also helps you to ensure that the issue is caused by the library and not by other 3rd party code.
